<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtRunnerApp</class>
 <widget class="QMainWindow" name="QtRunnerApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtRunnerApp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QPushButton" name="add_button">
            <property name="text">
             <string>Добавить нового сотрудника</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="remove_button">
            <property name="text">
             <string>Удалить сотрудника</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QTreeWidget" name="treeWidget">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QPushButton" name="create_employee_button">
            <property name="text">
             <string>Сохранить</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="close_employee_button">
            <property name="text">
             <string>Закрыть</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="5" column="0">
           <layout class="QHBoxLayout" name="positionHBox">
            <item>
             <widget class="QLabel" name="positionLabel">
              <property name="text">
               <string>Должность</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="positionLineEditor_2"/>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <layout class="QHBoxLayout" name="patronymicHBox">
            <item>
             <widget class="QLabel" name="patronymicLabel">
              <property name="text">
               <string>Отчество</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="patronymicLineEdit"/>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="nameHBox_2">
            <item>
             <widget class="QLabel" name="nameLabel">
              <property name="text">
               <string>Имя</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="nameLaneEditor"/>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="nameHBox">
            <item>
             <widget class="QLabel" name="surnameLabel">
              <property name="text">
               <string>Фамилия</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="surnameLineEditor"/>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <layout class="QHBoxLayout" name="age_HBox">
            <item>
             <widget class="QLabel" name="ageLabel">
              <property name="text">
               <string>Возраст</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ageLineEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
